lib/v2/datasource/remote/model/rate_model.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:equatable/equatable.dart';
+
+class RateModel extends Equatable {
+  final double seedsPerUSD;
+  final bool error;
```

Comment:
another error inside the model. please remove.

Reply:
Done too 👍